test/bondingcurve/EthBondingCurve.test.js

```diff
@@ -22,6 +24,15 @@ describe('EthBondingCurve', function () {
   let beneficiaryAddress1;
   let beneficiaryAddress2;
 
+  this.beforeAll(async function() {
+    // Can only get the current price on a forked network (since we haven't deployed Uniswap stuff in test setup)
+    if (!hre.network.config.forking) {
```

Comment:
This will execute by default if someone has a mainnet key provided in .env. Can we also add a flag to turn off forking?

Reply:
Done.